frontend/src/components/contexts/contexts.ts

```diff
@@ -0,0 +1,11 @@
+import * as React from "react";
+import { MockProfileBackendService } from "../../services/mock_profile_backend";
+import { ProfileHandlerService } from "../../services/profile_handler_service";
+
+const NUM_PROFILES = 10;
+
+export const MockBackendService = new MockProfileBackendService(NUM_PROFILES);
+export const ServiceHandlers = { 
```

Comment:
nit: Kind of a confusing name, maybe `defaultServices` or something? Also consts should almost always start with a lowercase. Some React structures like components and contexts are exceptions (so `ServiceContext` can stay how it is)